sky/provision/runpod/instance.py

```diff
@@ -89,7 +89,10 @@ def run_instances(region: str, cluster_name_on_cloud: str,
                 disk_size=config.node_config['DiskSize'],
                 image_name=config.node_config['ImageId'],
                 ports=config.ports_to_open_on_launch,
-                public_key=config.node_config['PublicKey'])
+                public_key=config.node_config['PublicKey'],
+                preemptible=config.node_config['Preemptible'],
+                bid_per_gpu=config.node_config['bid_per_gpu'],
```

Comment:
Should we keep the same naming style, i.e. `BidPerGPU`?

Reply:
Good catch! Updated!